// Wrapper para lidar com erros assÃ­ncronos
module.exports = fn => {
    return (req, res, next) => {
        fn(req, res, next).catch(next);
    };
};
